scripts/run.sh

```diff
@@ -18,14 +20,22 @@ fi
 
 case $1 in
 
-  legacy_events)
-    python -m pipe_events
-    ;;
-
   gap_events)
     ${THIS_SCRIPT_DIR}/gap_events.sh "${@:2}"
     ;;
 
+  encounter_events)
+    ${THIS_SCRIPT_DIR}/encounter_events.sh "${@:2}"
+    ;;
+
+  anchorage_events)
+    ${THIS_SCRIPT_DIR}/anchorage_events.sh "${@:2}"
+    ;;
+
+  fishing_events)
+    ${THIS_SCRIPT_DIR}/fishing_events.sh "${@:2}"
+    ;;
+
```

Comment:
in the else `*)` change `exit 0` with `exit 1`. This is kind of a virus it is spreading everywhere :S ... the problem is that airflow may get a fall success when it is actually displaying the correct usage of the command.